assets/src/edit-story/components/keyboard/index.js

```diff
@@ -69,6 +69,9 @@ function useKeyEffectInternal(
   const batchingCallback = useBatchingCallback(callback, deps || []);
   useEffect(
     () => {
+      if (!refOrNode) {
+        return undefined;
```

Comment:
In some cases the `refOrNode` is null and has no `current`, this is a guard against that.